### PR TITLE
AVRO-4000 Upgrade parent pom to 33

### PR DESCRIPTION
**Pull request by slachiewicz**
## What is the purpose of the change

Upgrade Parent POM to accommodate plugin updates

## Verifying this change

This change is a trivial cleanup without any test coverage.

## Documentation

- Does this pull request introduce a new feature? no

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,23 +47,19 @@
     <avro.docDir>build/avro-doc-${project.version}/api</avro.docDir>
 
     <!-- plugin versions -->
-    <apache-rat-plugin.version>0.16.1</apache-rat-plugin.version>
```
Comment (Fokko) [inline pom.xml:50]: Are these renames required? People might be overriding specific properties, and now we change them :)

Comment (jbonofre) [reply pom.xml:50]: I don't think renaming properties is required. I would rather remove the properties (when an override is not needed) to use the depManagement/pluginManagement from the parent.

**Review by jbonofre — CHANGES_REQUESTED** [review]

**Comment by slachiewicz** [comment]
I've reused properties from the parent and left overwrites in places where the previously used version was never than in the parent. 
If we agree that downgrades are ok, I can remove a few more properties

**Comment by slachiewicz** [comment]
Renaming properties were required in order to use dependency management from the parent. 
I could keep old property names and then remap it from old to new - but this would make pom more verbose

**Comment by jbonofre** [comment]
My point is that we can use the parent version without using the properties (directly implicit version from the parent). It's what we do in most of the Apache projects.

**Comment by slachiewicz** [comment]
I've updated to use our recently released Parent POM 33 and rebased.

**Review by opwvhk — APPROVED** [review]

**Review by martin-g — APPROVED** [review]

**Pull request closed without merging**